docs/current_docs/manuals/developer/runtimes.mdx

```diff
@@ -17,7 +21,7 @@ For now, the TypeScript SDK only supports Node.js (stable) and Bun (experimental
 
 To change the TypeScript SDK runtime, set the field `dagger.runtime` in the `package.json` file.
```

Comment:
Can we specify which package.json we mean, are we talking about the project level one or the one that we generate inside of `dagger/package.json` when running `init --sdk=typescript`

Reply:
It's the one at the root of the repo, not in `sdk/` since it's ignored by vsc

Reply:
Added, please check